packages/runner-ct/src/lib/state.js

```diff
@@ -187,6 +186,7 @@ export default class State {
   }
 
   runMultiMode = async () => {
+    const eventManager = await import(/* webpackChunkName: "ctChunk-EventManager" */'./event-manager').then((module) => module.default)
```

Comment:
I am not super familiar w/ code splitting - what exactly is the benefit of naming this chunk here?

Reply:
1) – we have a readable file name in the system
2) - we can ensure that we handle only our file systems